setup/docker/01_setup_enketo.bash

```diff
@@ -17,23 +17,33 @@ SHA1SUM_CONFIG_FILE_PATH="${CHECKSUM_DIR_PATH}/${CONFIG_FILE}.sha1"
 LAST_BUILD_COMMIT_FILE="last_build_commit.txt"
 LAST_BUILD_COMMIT_FILE_PATH="${CHECKSUM_DIR_PATH}/${LAST_BUILD_COMMIT_FILE}"
 
+RUN_GRUNT=0
 # Compare config version
-set +e  # Do not exit on `sha1sum` or `cat` error
-sha1sum -c "${SHA1SUM_CONFIG_FILE_PATH}"
-RUN_GRUNT=$([[ $? -eq 0 ]] && echo 0 || echo 1)
+sha1sum -c "${SHA1SUM_CONFIG_FILE_PATH}" || RUN_GRUNT=1
 
 # Compare commit version
 CURRENT_COMMIT=$(git rev-parse HEAD) # Get current commit
-LAST_BUILD_COMMIT=$(cat ${LAST_BUILD_COMMIT_FILE_PATH})
+LAST_BUILD_COMMIT=$(cat ${LAST_BUILD_COMMIT_FILE_PATH}) || echo "-"
```

Comment:
If you want `-` to end up in echo `LAST_BUILD_COMMIT` if there's a problem reading from `LAST_BUILD_COMMIT_FILE_PATH`, then I think the `|| echo "-"` should be inside the `$()`:
```
john@world$ LAST_BUILD_COMMIT_FILE_PATH=fake
john@world$ LAST_BUILD_COMMIT=$(cat ${LAST_BUILD_COMMIT_FILE_PATH}) || echo "-"
cat: fake: No such file or directory
-
john@world$ echo $LAST_BUILD_COMMIT

john@world$ LAST_BUILD_COMMIT=$(cat ${LAST_BUILD_COMMIT_FILE_PATH} || echo "-")
cat: fake: No such file or directory
john@world$ echo $LAST_BUILD_COMMIT
-
```

Reply:
Totally right ;-)